src/com/github/johnthagen/cppcheck/CppCheckInspectionImpl.java

```diff
@@ -45,72 +55,121 @@ private static ProblemHighlightType severityToHighlightType(@NotNull final Strin
+
+        final NodeList errors = doc.getElementsByTagName("error");
+        for (int i = 0; i < errors.getLength(); ++i)
+        {
+            /*
+                <error id="accessMoved" severity="warning" msg="Access of moved variable &apos;a&apos;." verbose="Access of moved variable &apos;a&apos;." cwe="672" hash="6576707224072251515" inconclusive="true">
+                    <location file="/mnt/s/clion/example_lite_2/test.cpp" line="14" column="18" info="Access of moved variable &apos;a&apos;."/>
+                    <location file="/mnt/s/clion/example_lite_2/test.cpp" line="13" column="7" info="Calling std::move(a)"/>
+                    <symbol>a</symbol>
+                </error>
+            */
+
+            final Node error = errors.item(i);
+            final NamedNodeMap attrs = error.getAttributes();
+
+            final String id = attrs.getNamedItem("id").getNodeValue();
+
+            // Skip the "toomanyconfigs" error
+            /*
+       			<error id="toomanyconfigs" severity="information" msg="Too many #ifdef configurations - cppcheck only checks 1 of 12 configurations. Use --force to check all configurations." verbose="The checking of the file will be interrupted because there are too many #ifdef configurations. Checking of all #ifdef configurations can be forced by --force command line option or from GUI preferences. However that may increase the checking time." cwe="398">
+                    <location file="C:\Users\Quotenjugendlicher\AppData\Local\Temp\___valueflow.cpp" line="0" column="0"/>
+                </error>
+            */
+            if (id.equals("toomanyconfigs")) {
+                continue;
+            }
+
+            // Skip the "missingIncludeSystem" error
+            /*
+       			<error id="missingIncludeSystem" severity="information" msg="Cppcheck cannot find all the include files (use --check-config for details)" verbose="Cppcheck cannot find all the include files. Cppcheck can check the code without the include files found. But the results will probably be more accurate if all the include files are found. Please check your project&apos;s include directories and add all of them as include directories for Cppcheck. To see what files Cppcheck cannot find use --check-config."/>
+            */
+            if (id.equals("missingIncludeSystem")) {
+                continue;
+            }
+
+            final String severity = attrs.getNamedItem("severity").getNodeValue();
+            final String errorMessage = attrs.getNamedItem("msg").getNodeValue();
+            final Node inconclusiveNode = attrs.getNamedItem("inconclusive");
+            final boolean inconclusive = inconclusiveNode != null && inconclusiveNode.getNodeValue().equals("true");
+
+            Node location = null;
+
+            // look for the first "location" child name
+            final NodeList children = error.getChildNodes();
+            for(int j = 0; j < children.getLength(); ++j)
```

Comment:
Could you run the default IntelliJ formatter on code prior to commit? (`Ctrl+Alt+L`) Thanks.

Reply:
Done. Also renamed some variables to match the existing style.